### ISSUE TITLE
achievement for playing the game 24 hours straight without going afk for more than 1 hour.

### ISSUE THREAD

**Issue by isavailablh** [studio-minus/ppg-bugs-and-requests#1551]
**Feature description**
Getting an achievement called "Serious dedication" for playing the game for 24 hours straight and not going AFK for more than 1 hour.

**Feature purpose**
Makes people feel good about themselves.

**Additional context**
Nothing else. Call it serious dedication.

**Comment by DindinYT37** [comment]
How would the game know you aren't afk?

**Comment by bibbybabby** [comment]
> How would the game know you aren't afk?

Same way it knows you aren't afk in Pacifist

**Comment by DindinYT37** [comment]
> > How would the game know you aren't afk?
> 
> Same way it knows you aren't afk in Pacifist

Right, that existed. Though it is arguable if the way it checks if you're afk for Pacifist is good.

**Comment by isavailablh** [comment]
i think this is like a really good achievement. My other one achievement for being me is stupid and a joke my friend told me to do otherwise id lose 1000 bucks but this one is actually good. Especially cuz i messed up pacifist 3 times with the 24 hours and never got any achievement. It would be cool if this existed.
